docs/pages/radiogroup.js

```diff
@@ -0,0 +1,986 @@
+        />
+        <MainSection.Subsection
+          title="Legends"
+          description={`Each RadioGroup should have a \`legend\` that clearly delineates what is being chosen. If you cannot use the provided legend styling, \`legendDisplay\` can be set to \`hidden\`, and an alternative legend can be displayed. See the [legend visibility](#Legend-visibility) variant for an example.`}
+        />
+        <MainSection.Subsection
+          title="Keyboard interaction"
+          description={`After focus has been set on the first RadioButton inside a RadioGroup, the arrow keys are used to cycle focus between the various options. Clicking or tapping the label of RadioButton should also focus that particular RadioButton. All RadioGroup.RadioButtons within a RadioGroup should share the same \`name\` to ensure keyboard accessibility, but that \`name\` needs to be unique from other RadioGroup buttons on the page.`}
+        />
+        <MainSection.Subsection
+          title="Error message"
+          description={`If RadioGroup has an \`errorMessage\`, the \`id\` on RadioGroup must also be set to properly link the error message to RadioGroup.
+    `}
+        />
+      </MainSection>
+      <MainSection
+        name="Localization"
+        description={`Be sure to localize \`errorMessage\`, \`subtext\`, \`label\`, and \`legend\`. Be mindful of label length so that it doesn’t truncate in languages with lengthier character counts.`}
+      />
+
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          title="Direction"
+          description="RadioGroups can be shown in a column or row by specifying the `direction` property."
+        >
+          <MainSection.Card
+            defaultCode={`
+function RadioButtonExample() {
+  const [favorite, setFavorite] = React.useState();
+  const [favoriteFood, setFavoriteFood] = React.useState();
+
+  return (
+    <Flex gap={8}>
+      <RadioGroup legend="What is your favorite pet?" id="directionExample-1">
+        <RadioGroup.RadioButton
+          checked={favorite === 'dogs'}
+          id="favoriteDog"
+          label="Dogs"
+          name="favorite"
+          onChange={() => setFavorite('dogs')}
+          value="dogs"
+        />
+        <RadioGroup.RadioButton
+          checked={favorite === 'cats'}
+          id="favoriteCat"
+          label="Cats"
+          name="favorite"
+          onChange={() => setFavorite('cats')}
+          value="cats"
+        />
+        <RadioGroup.RadioButton
+          checked={favorite === 'plants'}
+          id="favoritePlants"
+          label="Plants"
+          name="favorite"
+          onChange={() => setFavorite('plants')}
+          value="plants"
+        />
+      </RadioGroup>
+
+      <RadioGroup legend="What is your favorite snack?" errorMessage="Please select one" direction="row" id="directionExample">
```

Comment:
Should we use the recommended error message? 

`To continue you must select one item from this list`

Reply:
Given it's not in a flow, I think Please select one is fine